discovery_engine_flutter/example/ios/Runner/Info.plist

```diff
@@ -43,5 +43,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<true/>
+	<key>CADisableMinimumFrameDurationOnPhone</key>
+	<true/>
```

Comment:
This prevents the example app to be limited to 60Hz, it seems that scrolling on a 120Hz iPhone can feel laggy without that option(?).

It either way only affects the example.

My guess is it was auto-added by some tool.